backends/ebpf/CMakeLists.txt

```diff
@@ -221,6 +221,36 @@ else()
   set (SUPPORTS_KERNEL False)
 endif()
 
+# List of executable dependencies
+set(PROGRAMS scapy tcpdump pcap)
+
+# List of library dependencies
+set(LIBRARIES libpcap-dev gcc-multilib)
+
+# Check for each program
+foreach(PROG ${PROGRAMS})
+  find_program(${PROG}_FOUND ${PROG})
+  if (${PROG}_FOUND)
+    message(STATUS "Found program ${PROG} at ${${PROG}_FOUND}")
```

Comment:
should this be in a macro in common code, so it does not need to be duplicated in each backend?  We already have a lot of cruft in our cmake files that is a result of people blindly copying things around and duplicating things without really understanding how the cmake build process works.  We need to clean things up and definitely don't want to make it worse.

Reply:
I understand your concern. I have created a macro so that it becomes scalable across different backends.